doc/advanced/composing_routers.md

```diff
@@ -405,9 +405,53 @@ All the beer-routes now match in constant time.
 |-----------------|---------|-----------------------
 | `/beers/sahti`  | 40ns    | static
 
+### Wrapping a swappable ring handler
+
+In order for a ring handler to be recomposed, we can wrap it into a handler that dereferences it on request.
+
+```clj
+(defn deref-handler [rf]
```

Comment:
this could be just:

```clojure
(defn deref-handler [rf]
  (reitit.ring/reloading-ring-handler (fn [] @rf)))
```

, but I think your example is simpler.